envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/EnvoyAssertions.kt

```diff
@@ -40,7 +40,8 @@ fun ObjectAssert<EnvoyContainer>.hasOneAccessDenialWithActionBlock(
     method: String,
     clientName: String,
     trustedClient: Boolean,
-    clientIp: String
+    clientIp: String,
+    clientAllowedToAllEndpoints: Boolean = false
```

Comment:
this variable name feels weird to me, maybe 'clientAllowedOnAllEndpoints' or sth like that would feel more natural?

Reply:
I read `clientAllowedToAllEndpoints` as client allowed to *access* all endpoints. I guess @lukidzi did not want the name to be too long.